docs/source/en/model_doc/propainter.md

```diff
@@ -120,6 +120,13 @@ inputs = image_processor(images = video, masks = masks)
 import transformers as t
 model = t.ProPainterModel.from_pretrained("ruffy369/ProPainter")
 
+#For inference and getting that flashy final output(can be commented out)
+model.eval()
```

Comment:
FYI from_pretrained puts a model in evaluation mode by default, so this line can be omitted

Reply:
@NielsRogge thank you for mentioning. I changed the code in the latest commits